tests/sandbox/test_sandbox_application.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from pytest_check import check
+import requests
+import json
+
+
+from data import Actor
+from typing import Callable
+from utils import HttpMethod
+
+
+@pytest.mark.sandbox
+class TestSandboxApplication:
+    @pytest.mark.parametrize("actor", Actor)
+    def test_get_codesystem(
+        self,
+        load_json: Callable[[str], str],
+        send_rest_request: Callable[[HttpMethod, str, Actor], requests.Response],
+        actor: Actor,
+    ):
+        codeSystemType = "SPECIALTY"
+        response = send_rest_request(
+            HttpMethod.GET, f"FHIR/STU3/CodeSystem/{codeSystemType}", actor
+        )
+
+        with check:
+            assert response.status_code == 200, (
+                f"UNEXPECTED RESPONSE: "
```

Comment:
Does the lint not complain about an f string without any arguments?

Reply:
Hmm it doesn't seem to but this certainly doesn't need to be a format string however so have removed it.